scripts/intfutil

```diff
@@ -149,6 +150,32 @@ def appl_db_port_status_get(appl_db, intf_name, status_type):
         status = ','.join(new_speed_list)
     return status
 
+def state_db_port_status_get(db, intf_name, field):
+    """
+    Get the port status
+    """
+    full_table_id = PORT_STATE_TABLE_PREFIX + intf_name
+    status = db.get(db.STATE_DB, full_table_id, field)
+    if not status:
+        return "N/A"
+    if field == PORT_SPEED and status != "N/A":
+        speed = int(status)
+        if speed >= 1000:
+            status = '{}G'.format(int(speed / 1000))
+        else:
+            status = '{}M'.format(speed)
+    elif field in [PORT_ADV_SPEEDS, PORT_RMT_ADV_SPEEDS] and status not in ["N/A", "all"]:
```

Comment:
I don't think we have PORT_ADV_SPEEDS field in STATE_DB. Why are we adding check here?

Reply:
Yes, you're right, it's removed now